blocks.md

```diff
@@ -62,17 +62,21 @@ critical_updates_regex = "(linux |linux-lts|linux-zen)"
 Key | Values | Required | Default
 ----|--------|----------|--------
 `interval` | Update interval in seconds. | No | `600`
-`format` | A string to customise the output of this block. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{count}"`
-`format_singular` | Same as `format`, but for when exactly one update is available. | No | `"{count}"`
-`format_up_to_date` | Same as `format`, but for when no updates are available. | No | `"{count}"`
+`format` | A string to customise the output of this block. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{count:1}"`
```

Comment:
Can you add something like "Refer to [formatting documentation](blocks.md#formatting) to customize formatting strings' placeholders."  in the description for `format`?

Reply:
Do you mean to add this reference for every block? Given the direction of moving all the blocks to formatting strings, I don't think it's a good idea, because it is just too redundant. Maybe add a note in the beginning of blocks.md?

Reply:
Yeah I meant to add it to every block like the Escaping Text reference, however the only reason is for visibility. Moving it to the beginning of blocks.md could be OK too. In the future it might be best to overhaul the docs, especially for top-level config stuff.